src/types/onyx/TransactionViolation.ts

```diff
@@ -6,33 +6,77 @@ import type CONST from '@src/CONST';
  */
 type ViolationName = (typeof CONST.VIOLATIONS)[keyof typeof CONST.VIOLATIONS];
 
+/** Model of a transaction violation */
 type TransactionViolation = {
+    /** Type of transaction violation ('violation', 'notice', 'warning', ...) */
     type: string;
+
+    /** Name of the transaction violation */
     name: ViolationName;
+
+    /** Additional violation information to provide the user */
     data?: {
+        /** Who rejected the transaction */
         rejectedBy?: string;
+
+        /** Why the transaction was rejected */
         rejectReason?: string;
+
+        /** Limit that the transaction violated */
         formattedLimit?: string;
+
+        /** Percentage amount of conversion surcharge applied to the transaction */
         surcharge?: number;
+
+        /** Percentage amount of invoice markup applied to the transaction */
         invoiceMarkup?: number;
+
+        /** Amount of days which the transaction date overpasses the date limit */
         maxAge?: number;
+
+        /** Name of the tag that triggered this violation */
         tagName?: string;
+
+        // TODO: Doesn't seem to be used in app
         categoryLimit?: string;
+
+        // TODO: Doesn't seem to be used in app
         limit?: string;
```

Comment:
Can you verify if `TransactionViolation` has these properties? If they exist you can add a comment with explanation